crates/physical-plan/src/plan.rs

```diff
@@ -1918,40 +1854,132 @@ mod tests {
             None,
         ));
 
-        let db = SchemaViewer {
-            schemas: vec![t.clone()],
-        };
-
-        let compile = |sql| {
-            let stmt = parse_and_type_sql(sql, &db).unwrap();
-            let Statement::Select(select) = stmt else {
-                unreachable!()
-            };
-            compile_select_list(select).optimize().unwrap()
-        };
-
-        let plan = compile("select * from t limit 5");
-
-        assert!(matches!(
-            plan,
-            ProjectListPlan::Name(ProjectPlan::None(PhysicalPlan::TableScan(
-                TableScan { limit: Some(5), .. },
-                _
-            )))
-        ));
+        let db = SchemaViewer::new(vec![t.clone()]);
+
+        check_query(
+            &db,
+            "SELECT * FROM t LIMIT 5",
+            expect![[r#"
+Limit: 5
+  -> Seq Scan on t
+  Output: t.x, t.y"#]],
```

Comment:
```suggestion
Limit: 5
  Output: (t: RowId (table: t))
  -> Seq Scan on t"#]],
```

I know this is a deviation from pg, but I find it much easier to read if the output comes before the input. In particular for left deep joins.

Reply:
It will be possible for everything except `cross joins` that need to traverse the child nodes to found the projections. So, I don't think is good to have 2 different outputs.

Reply:
I ended doing this request, so now is like you asked :)